libraries/AC_PrecLand/AC_PrecLand_Companion.cpp

```diff
@@ -59,4 +65,6 @@ void AC_PrecLand_Companion::handle_msg(mavlink_message_t* msg)
 
     _los_meas_time_ms = AP_HAL::millis();
     _have_los_meas = true;
+    
+    _target_num = packet.target_num;
```

Comment:
Nitpicking: it would be nice this was together with the other assignments (below distance seems a good place).